web/e2e_test.go

```diff
@@ -964,6 +962,8 @@ func TestHealthz(t *testing.T) {
 }
 
 func TestRPC(t *testing.T) {
+	t.Skip("TODO grpc and jsonrpc")
```

Comment:
Are you planning to update this test before merging?

Reply:
No, I can just remove the test but was hoping to stumble on this in the future and then improve it.

Reply:
Let's remove it if we think it's not critical and have decent coverage elsewhere!